prisma/migrations/20210724193021_init/migration_002.sql

```diff
@@ -0,0 +1,10 @@
+-- Add scouting columns to users table
```

Comment:
Not sure exactly what format you need migrations to be provided in, but these are the new columns.

Reply:
Not the right way of making this but understandable as it's new! You want to add the tables to the `prisma.schema` file which is the source of truth for the database and then run `prisma migrate dev --name add_national_team_scouting` and this will generate the migration for you in the right place and way :) Here's Prisma's documents which will hopefully help

https://www.prisma.io/docs/concepts/components/prisma-migrate

Reply:
Okayyyy, migration added! Thanks. 😊